.travis.yml

```diff
@@ -48,6 +48,23 @@ jobs:
       script:
         - make cephfsplugin
 
+after_script:
+
+# Make root mounted as rshared to fix kube-dns issues.
+- sudo mount --make-rshared /
+# Download kubectl, which is a requirement for using minikube.
+- curl -Lo kubectl https://storage.googleapis.com/kubernetes-release/release/v1.13.0/bin/linux/amd64/kubectl && chmod +x kubectl && sudo mv kubectl /usr/local/bin/
```

Comment:
I would stick to 1.13.5 as the Kubernetes version here.

Reply:
Done.